CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.1.63
+
+* updated `unawaited_futures` to ignore assignments within cascades
+* new lint: `sort_dependencies`
```

Comment:
I think it was renamed to "sort_pub_dependencies".

Reply:
> I think it was renamed to "sort_pub_dependencies".

Yes